Source/SwiftLintFramework/Rules/CommaRule.swift

```diff
@@ -50,18 +50,18 @@ public struct CommaRule: CorrectableRule, ConfigurationProviderRule {
     }
 
     public func correctFile(file: File) -> [Correction] {
-        let matches = violationRangesInFile(file)
+        let violations = violationRangesInFile(file)
+        let matches = file.ruleEnabledViolatingRanges(violations, forRule: self)
         if matches.isEmpty { return [] }
 
         var contents = file.contents as NSString
```

Comment:
yeah, that needs to be NSString because `stringByReplacingCharactersInRange` doesn't show up on String.